checks/runtime.go

```diff
@@ -138,17 +137,17 @@ func CheckSensitiveDirs(client *client.Client) (res Result) {
 	return
 }
 
-func CheckSSHRunning(client *client.Client) (res Result) {
+func CheckSSHRunning(t actuary.Target) (res Result) {
 	var badContainers []string
 	res.Name = "5.6 Do not run ssh within containers"
-	containers := actuary.CreateContainerList(client)
+	containers := t.Containers
 
 	if !containers.Running() {
 		res.Skip("No running containers")
 		return
 	}
 	for _, container := range containers {
-		procs, _ := client.ContainerTop(container.ID, []string{})
+		procs, _ := t.Client.ContainerTop(container.ID, []string{})
```

Comment:
Should we call `t.Client` here? We might get inconsistent results (the container might not even be running at this point).

`actuary.Target` should have all the info needed for all the tests, but I'm not sure if it will be harder to add new tests if we follow that strictly.